qlty-plugins/plugins/scripts/fetchLatestVersion/python.ts

```diff
@@ -10,8 +10,8 @@ export async function fetchLatestVersionForPython(
     throw new Error(`Failed to fetch from PyPI, status: ${response.status}`);
   }
 
-  const json = await response.json();
-  const versionString = json.info?.version as string;
+  const json = (await response.json()) as { info?: { version?: string } };
```

Comment:
I'd consider it optional for this PR, but this is the type of thing I would use Zod for. With Zod you get parsing, clear errors on unexpected data, and accurate types at a low cost.